nomenclature/definitions/variable/transport_types.yaml

```
@@ -0,0 +1,63 @@
+# This list of fuels will replace any occurrence of `<Transport>`
+# in a variable-definition yaml file
+
+<Transport>:
+  Maritime:
+     description: Maritime transportation technologies
+  
+  Maritime|Biofuels:
+     description: Maritime biofuels-based transportation technologies
+  
+  Maritime|LNG:
+     description: Maritime lng-based transportation technologies
+  
+  Maritime|Oil:
+     description: Maritime oil-based transportation technologies
+  
+  Rail:
+     description: Rail-based transportation technologies
+  
+  Rail|Electric:
+     description: Rail-based transportation technologies using electricity
+  
+  Rail|Oil:
+     description: Rail-based transportation technologies using oil products
+  
+  Road:
+     description: Road-based transportation technologies
+  
+  Road|Electric:
+     description: Road-based transportation technologies using electricity
+  
+  Road|Electric|BEV:
+     description: Road-based transportation technologies using battery electric vehicles
+  
+  Road|Electric|FCEV:
+     description: Road-based transportation technologies using fuel cell electric vehicles
+  
+  Road|Electric|Overhead:
+     description: Road-based transportation technologies using electricity via overhead trolley trucks
+  
+  Road|PHEV:
```

How would a user report how much of the service was delivered with the battery vs. ICE...? Or is that not a relevant question?
```suggestion
  Road|Plugin Hybrid:
```